src/DDTrace/Data/SpanContext.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace DDTrace\Data;
+
+use DDTrace\Contracts\SpanContext as SpanContextInterface;
+
+abstract class SpanContext implements SpanContextInterface
+{
+    /**
+     * The unique integer (64-bit unsigned) ID of the trace containing this span.
+     * It is stored in hexadecimal representation.
```

Comment:
This is unsigned 63 bits: [link](https://github.com/DataDog/dd-trace-php/blob/1665e91a9ed9717f2c5906c9a11e27981620a862/src/DDTrace/ID.php#L12).

Also, I don't think that this is stored as hexadecimal. The [generate id](https://github.com/DataDog/dd-trace-php/blob/1665e91a9ed9717f2c5906c9a11e27981620a862/src/DDTrace/ID.php#L12) is the string representation of a 63-bit unsigned integer number `"1234567"` and it is [stored as it is](https://github.com/DataDog/dd-trace-php/blob/1665e91a9ed9717f2c5906c9a11e27981620a862/src/DDTrace/SpanContext.php#L37)

Reply:
👍